lib/board.rb

```diff
@@ -0,0 +1,92 @@
+class Board
+
+    def initialize()
+        @grid = Array.new(3){Array.new(3, '_')}
+    end
+
+    def get_grid
+        @grid
+    end
+
+    def find_winner()
+
+        # returns row winner
+        @grid.each do |row|
+            row_winner = find_winner_in_line(row)
+            if row_winner
+                return row_winner
+            end
+        end
+
+        # returns column winner
+        @grid.transpose.each do |col|
+            col_winner = find_winner_in_line(col)
+            if col_winner
+                return col_winner
+            end
+        end
+
+        # returns diagonal winner
+        diagonal_winner = find_winner_in_diagonal(@grid)
+        if diagonal_winner
+            return diagonal_winner
+        end
+
+        # check for any unfilled space
+        @grid.each do |row|
+            row.each do |col|
+                if col == '_'
+                    return false
+                end
+            end
+        end
+
+        :tie
+    end
+
+    def display_board(console)
+        @grid.each do |row|
+            console.puts row.join(" ")
+        end
+    end
+
+    def game_over?()
+        if (@grid[0][0] == "X")
+            return true
+        end
+        false
+    end
```

Comment:
The logic of this seems incomplete.
Is it a WIP?

Reply:
It was an example when we started building the game, i think we have to remove it as i think it's not necessary. Will confirm with @kkrull

Reply:
Yup - this looks like it can be safely removed.